src/sequences/defaultRelease.js

```diff
@@ -45,14 +55,8 @@ export const defaultReleaseSequence = [
   cook.createReleaseNotes('avatar'),
   cook.byBranch,
   {
-    master: [
-      cook.tagCurrentCommit,
-      cook.pushTagToRemote,
-      cook.byReleaseTarget,
-      {
-        github: cook.createGithubRelease,
-      },
-    ],
+    master: publishToGithub,
+    main: publishToGithub,
```

Comment:
... and this other reference here.

@henri-hulski This is all we need. For the rest, we detect current release with tags (which has always seemed safer).